modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/Instructions.kt

```diff
@@ -173,6 +176,26 @@ object Instructions {
         }
     }
 
+    fun transferAsset(sourceId: AssetId, value: UInt, destinationId: AssetId): Instruction {
+        return transferSome(
+            IdBox.AssetId(sourceId),
+            Value.U32(value),
+            IdBox.AssetId(destinationId)
+        )
+    }
+
+    fun doIf(condition: Boolean, then: Instruction, otherwise: Instruction): Instruction {
```

Comment:
I think better use name of instruction `if` as is, without `do` even the word is reserved by language. Let it be framed by `